### PR TITLE
Add hasHeader in Response

### PR DESCRIPTION
**Pull request by niden**
Hello!

* Type: new feature
* Link to issue: [PR-12189](https://github.com/phalcon/cphalcon/pull/12189)

**In raising this pull request, I confirm the following (please check boxes):**

- [x] I have read and understood the [Contributing Guidelines](https://github.com/phalcon/cphalcon/blob/master/CONTRIBUTING.md)?
- [x] I have checked that another pull request for this purpose does not exist.
- [x] I wrote some tests for this PR.

Added `hasHeader` in `Phalcon\Http\Response` to allow checking for a valid header existence

Thanks

### CODE REVIEW

**Comment by niden** [comment]
cc @sergeyklay

File: CHANGELOG-4.0.md [hunk 1]
```diff
@@ -11,24 +11,27 @@
...
 - Changed `Phalcon\Mvc\Model` to use the `Phalcon\Messages\Message` object for its messages [#13114](https://github.com/phalcon/cphalcon/issues/13114)
 - Changed `Phalcon\Validation\*` to use the `Phalcon\Messages\Message` object for its messages [#13114](https://github.com/phalcon/cphalcon/issues/13114)
-- Collections now use the Validation component [#12376](https://github.com/phalcon/cphalcon/pull/12376)
+- Collections now use the Validation component [PR-12376](https://github.com/phalcon/cphalcon/pull/12376)
+- Refactored `Phalcon\Db\Adapter\Pdo::query` to use PDO's prepare and execute. `Phalcon\Db\Adapter::fetchAll` to use PDO's fetchAll
 - Fixed  `\Phalcon\Http\Response::setFileToSend` filename last much _ 
+- Changed `Phalcon\Tag::getTitle()`. It returns only the text. It accepts `prepend`, `append` booleans to prepend or append the relevant text to the title. [13547](https://github.com/phalcon/cphalcon/issues/13547)
```
Comment (sergeyklay) [inline CHANGELOG-4.0.md]:
```suggestion
- Changed `Phalcon\Tag::getTitle()`. It returns only the text. It accepts `prepend`, `append` booleans to prepend or append the relevant text to the title. [#13547](https://github.com/phalcon/cphalcon/issues/13547) 
```

File: phalcon/http/response/headers.zep [hunk 2]
```diff
@@ -30,6 +30,20 @@ class Headers implements HeadersInterface
...
 	protected _headers = [];
 
+	/**
+	 * Sets a header to be sent at the end of the request
+	 */
+	public function has(string name) -> boolean
+	{
+		var header;
+
+		if !fetch header, this->_headers[name] {
```
Comment (sergeyklay) [inline phalcon/http/response/headers.zep]: `return isset(this->_headers[name]);` ?

**Review by sergeyklay — DISMISSED** [review]
LGTM (except minor things)

**Pull request merged**